static/js/admin.js

```diff
@@ -655,6 +655,12 @@ function _setup_page() {
             error: function (xhr, error) {
                 $('#admin-emoji-status, #admin-emoji-name-status, #admin-emoji-url-status').hide();
                 var errors = JSON.parse(xhr.responseText).msg;
+                if (typeof errors === "string" && errors === "Enter a valid URL.") {
```

Comment:
I think we'll probably end up wanting to change how the backend works since I think this won't handle i18n properly.